### PR TITLE
Add standard files

### PR DESCRIPTION
**Pull request by zippolyte**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| `invalid`                     | Issue Only               | Invalid issue.
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
+| `stale/exempt`                | Issues and Pull Requests | Exempt from being marked as stale.
+
```
Comment (gzussa) [inline CONTRIBUTING.md]: Let's remove some breaklines

File: LICENSE-3rdparty.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+Component,Origin,License,Copyright
```
Comment (gzussa) [inline LICENSE-3rdparty.csv]: Is this empty because it is not filled or because there is nothing to add in there?

File: NOTICE [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Datadog datadog-agent
-Copyright 2017 Datadog, Inc.
+Datadog Cloud Foundry Buildpack
+Copyright 2017-Present Datadog, Inc.
```
Comment (gzussa) [inline NOTICE]: is it 2015 or 2017 the year this project started?

Comment (zippolyte) [reply NOTICE]: earliest commit is 2017

File: LICENSE-3rdparty.csv (hunk 2 above)
Comment (zippolyte) [reply LICENSE-3rdparty.csv]: Nothing to add

File: lib/redirect-logs.sh [hunk 4]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2015-Present Datadog, Inc.
```
Comment (gzussa) [inline lib/redirect-logs.sh]: Then let's update to 2017. See https://github.com/DataDog/datadog-cloudfoundry-buildpack/pull/72/files#r362529642

Comment (zippolyte) [reply lib/redirect-logs.sh]: 🤦‍♂

File: RELEASING.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Releasing
+
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+
+Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule of one minor release every 6 weeks.
```
Comment (zippolyte) [inline RELEASING.md]:
```suggestion
Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule (+1/2 weeks).
```

**Pull request merged**